tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs

```diff
@@ -861,6 +861,7 @@ public void SubsequentBuilds_CopyOutput_ToOutputDirectory_NestedUnderSourceDirec
 
         [Theory]
         [InlineData("github-actions")]
+        [InlineData("github-actions-buster")]
```

Comment:
Could you also test `github-actions-buster` image in Node and Ruby sample apps test if applicable.

Reply:
i can add one for node .. but github action doesn't have ruby .. i added tests for the new images whereever I saw a test for github-actions current image